### ISSUE TITLE
Pass User Information into VBMS claim service

### ISSUE THREAD

**Issue by shanear** [department-of-veterans-affairs/caseflow#5134]
We were informed that we should be passing the user info for who created the claim in the claim service.

Working with VBMS to see how to do this.

We committed to having this fixed by May 11, 2018.

**Comment by nicholasholtz** [comment]
For documentation purposes:

This requirement was identified in a call with BGS on April 11, 2018. 
Attendees:
1. Nathan Jolly (VBMS)
2. John Dell (BGS)
3. Shane Russell
4. Nicholas Holtz
5. Mike Edsall (AMO)
6. Josh Will (NWQ)

John Dell identified the fact that we are not passing the specific user when we establish the claim as presenting an auditing concern that could raise issues with the Inspector General.
- There are two different methods to get into the system to establish claims
  - one is what we are currently using which allows generic systems to establish claims, but there are fields we are bypassing that does not indicate who is doing the work
  - this is important because there are external actors, such as POA, that are capable of establishing claims; as such, it is required that the user be identified. 
- Right now, they cannot tell who is creating Caseflow-established claims.
  - We are using a generic user id
- Concern is that system could be audited
- Shane noted that we have this information for all claims we have and are establishing, it is just not passed over to BGS. 
- We were asked if we could get this in place within a month, which Shane acknowledged as doable.

**Comment by shanear** [comment]
This will be deployed into UAT tommorrow. Sent an email to John and Nate for how to proceed with testing until we launch it in prod.

**Comment by shanear** [comment]
so it turns out we are going to have to do some far more complicated crypto signature stuff in order to make this work.

**Comment by shanear** [comment]
The solution proposed to us by VBMS won't work. This might balloon into something even bigger 😑Waiting on the VBMS team to respond

**Comment by shanear** [comment]
I chatted with the VBMS team about this last week. They admitted that their solution was not optimal, and agreed to building a way for us to pass the user in. If they complete that, then we will update our code accordingly.

**Comment by leikkisa** [comment]
Closing in favor of #6938
